src/hyperResource.js

```diff
@@ -581,11 +585,15 @@ function getVersions( resource ) { // jshint ignore:line
 
 function getUrlFn( resources, prefix, version ) { // jshint ignore:line
 	var cache = getActionUrlCache( resources, prefix, version );
-	return function( resourceName, actionName, data, parentUrl, context ) {
-		return cache[ resourceName ][ actionName ]( data, parentUrl, context );
+	return function( resourceName, actionName, data, parentUrl, envelope ) {
+		return cache[ resourceName ][ actionName ]( data, parentUrl, envelope );
 	};
 }
 
+function isTemplated( url ) { // jshint ignore:line
+	return url.indexOf( "{" ) > 0 || url.indexOf( ":" ) > 0;
```

Comment:
Should these be looking for `> -1` or `>= 0` or are we trying to ignore the first character of the string?